bin/mdsctl

```diff
@@ -311,13 +311,17 @@ installImages() {
 }
 
 buildImage() {
-  docker build -t ${1}$(getBranchSuffix) .
+  docker build -t ${1}$(getBranchSuffix)$(getCommitHash) .
 }
 
 getBranchSuffix() {
   git rev-parse --abbrev-ref HEAD | sed -E -e 's/[^-a-zA-Z0-9]/-/g' -e 's/^/-/'
 }
 
+getCommitHash() {
+  git rev-parse HEAD | sed -E -e 's/[^-a-zA-Z0-9]/-/g' -e 's/^/-/'
```

Comment:
When I run this command I get
`-366bf96cc1bad92cc0b79c4a7951dde536208513`
I guess we want that `-` character at the beginning? Also, that's the full hash, not the shortened one that's normally shown with commits (which is probably fine).

Reply:
Yeup, that's intended indeed. Just serves as a delimiter in the image tag.